src/makeCalculator.js

```diff
@@ -37,7 +37,39 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  const calculator = {
+    add: (number) => {
+      calculator.result += number;
+
+      return calculator;
+    },
+    subtract: (number) => {
+      calculator.result -= number;
+
+      return calculator;
+    },
+    multiply: (number) => {
+      calculator.result *= number;
+
+      return calculator;
+    },
+    divide: (number) => {
+      calculator.result /= number;
+
+      return calculator;
+    },
+    reset: () => {
+      calculator.result = 0;
+
+      return calculator;
```

Comment:
```suggestion
      return this;
```

Reply:
tests don't pass after the proposed changes

Reply:
ok, let it be like this. It's better to use `this` for calculating as well and reorganize object a little. But your variant is also ok